_docs-2/getting-started/table_design.md

```diff
@@ -85,7 +85,7 @@ date range.
 Since Keys in Accumulo are sorted lexicographically by default, it's often useful to encode
 common data types into a byte format in which their sort order corresponds to the sort order
 in their native form. An example of this is encoding dates and numerical data so that they can
-be better seeked or searched in ranges.
+be better sought or searched in ranges.
```

Comment:
```suggestion
perform better during a seek or range search.
```

Reply:
Merged in 24b0eb6d124e844acad27f6222605af7171aea7a